sources/packages/forms/src/form-definitions/studentfinancialinformationappeal.json

```diff
@@ -279,7 +298,7 @@
                             "value": ""
                         }
                     ],
-                    "content": "If you have had, or anticipate having a significant decrease in income for {{data.calculatedTaxYear}}, you may request to be assessed using this year's estimated income.",
+                    "content": "If you have had, or anticipate having a significant decrease in income for {{data.currentTaxYear}}, you may request to be assessed using this year's estimated income.",
```

Comment:
Please use data.programYear, or please explain why this was not used.

Reply:
The `data.programYear` in this form is not actually the "2024-2025" program year name?
If I am right the question from @guru-aot would not make sense and and reply from @bidyashish should probably mention it. Please let me know if I am missing something.